model/Check/System/StuckTasksCheckService.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace oat\taoSystemStatus\model\Check\System;
```

Comment:
let's move that service from the cheks folder. For example model/SystemStatus

Reply:
done